features/steps/running_tests.py

```diff
@@ -14,3 +14,8 @@ def run_command(context):
 @then(u'I should see the behave tests run')
 def is_running(context):
     pass
+
+
+@then(u'context.django should be declared')
+def django_context(context):
+    assert context.django
```

Comment:
Is this a correct test? What if the code piece calling the hook is removed from the code? Then this won't be called and will not fail.

Reply:
Stupid me! This step is called by _behave_ in any case! :blush: So, this _is_ a correct test. :+1: